# preload the correct libgcc_s.so for libgfortran.so,
# which is required for numpy
#
--- scripts/xpra_launcher.orig	2019-03-19 15:05:07 UTC
+++ scripts/xpra_launcher
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 
 import os
+from ctypes import cdll
+cdll.LoadLibrary("%%GCC_RUNTIME%%/libgcc_s.so")
+
 import sys
 try:
     import xpra
